pcf/particle/aws/dynamodb/dynamodb_table.py

```diff
@@ -174,5 +187,9 @@ def is_state_definition_equivalent(self):
         self.get_state()
         current_definition = pcf_util.param_filter(self.current_state_definition, DynamoDB.REMOVE_PARAM_FILTER, True)
         desired_definition = pcf_util.param_filter(self.desired_state_definition, DynamoDB.START_PARAM_FILTER)
+        # self.create_table() does not return "Tags" as an attribute therefore, current_state_definition does not have
+        # any reference to Tags, so it is removed from the comparison between current_definition and desired_definition
+        if self.desired_state_definition.get("Tags"):
```

Comment:
doesn't look like update works with tags.

Reply:
The client.update_table() method does not take Tags as an attribute. I had implemented the feature to solely include Tags on the initial creation of the table. I can implement the updating of tags functionality if desired. It would be best to create another method say `update_table_tags()` which utilizes the boto3 `list_tags_of_resource()`, `tag_resource()` and `untag_resource` methods. Separation of concerns is important here because you don't need to update the entire table if you only want to update the table tags and vice versa. What do you think?

Reply:
^yeah I think thats the way to update tags for now until we implement the more generic way to handle tags for resources